torch/package/_selective_intern.py

```diff
@@ -0,0 +1,9 @@
+SELECTIVE_INTERN_LIST = {
+    "foo",
+}
+
+if "__torch_package__" in dir():
```

Comment:
this approach doesn't scale well for deeper submodules (e.g. it works for `torch.foo`, but will it work for `torch.quantization.fx`?)